app/controllers/person_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PersonController < ApplicationController
+  def index
+    @person = Person.all
+    render :index
+  end
+
+  def show
+    @id = params[:id]
+    @person = Person.find(@id)
+    render :show
+  end
+
+  def persons_tasks
```

Comment:
Based on the fact that you are in the persons controller, I think you can assume that the person is automatically associated and just use an action called tasks.